docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/LightGbm.cs

```diff
@@ -20,10 +20,10 @@ public static void Example()
 
             // Create the Estimator pipeline. For simplicity, we will train a small tree with 4 leaves and 2 boosting iterations.
             var pipeline = mlContext.Ranking.Trainers.LightGbm(
-                numLeaves: 4,
-                minDataPerLeaf: 10,
+                leafCount: 4,
+                minimumDataPerLeaf: 10,
```

Comment:
```suggestion
                MinExampleCountPerLeaf: 10,
```

Reply:
Should the arguments have the same name as FastTreeArguments?
In FastTree these arguments are called `numTrees` and `minDatapointsInLeaves`.

---
In reply to: [261456738](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261456738) [](ancestors = 261456738)

Reply:
So according to issue #2613 `numTrees` should change to `numberOfTrees` (or `numberOfIterations`, as long as we are consistent with FastTree).

---
In reply to: [261698972](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261698972) [](ancestors = 261698972,261456738)

Reply:
My preference is that they should follow what is in LightGBM, but in a more verbose form to align with what we have in C#. For example, num_iterations == NumberOfIterations, min_data_in_leaf == minimumDataInLeaf. This way they bear some resemblance of the LightGBM arguments if a user looks it up.


---
In reply to: [261699860](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261699860) [](ancestors = 261699860,261698972,261456738)

Reply:
Resolved to parameters that are the same between FastTree and LightGBM, they are named the same (NumberOfIterations, MinimumExampleCountPerLeaf).
Parameters that are LightGBM specific are closer or the same to LIghtGBM parameters but in their verbose form: MinimumSplitGain, FeatureFraction, etc.



---
In reply to: [261722064](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261722064) [](ancestors = 261722064,261699860,261698972,261456738)

Reply:
@yaeld@microsoft.com  Wei-Sheng has a PR for fast tree that is updating the parameters to match what along with the changes that we are doing here.

---
In reply to: [261698972](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261698972) [](ancestors = 261698972,261456738)